code/src/components/ThoughtForm.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+const ThoughtForm = ({ onFormSubmit, newThought, setNewThought }) => {
+  return (
+    <div className="thought-wrapper">
+      <form onSubmit={onFormSubmit}>
+        <label htmlFor="newThought">Type your thought</label>
+        <input className="inputfield" id="newThought" type="text" value={newThought} onChange={(e) => setNewThought(e.target.value)} maxLength="140" placeholder="Type Here...." />
+        <p className="wordcount-newthought">
+          <span className="newthought-color"> {newThought.length}</span>/ 140
+        </p>
+
+        <button className="ThoughtButton" disabled={newThought.length < 5} type="submit">
+          <span className="heart" role="img" aria-label="fonkelend hart">
+            💖
+          </span>{" "}
+          Send happy thought!
+          <span className="heart" role="img" aria-label="fonkelend hart">
```

Comment:
in the text-field you can use a widht and break line if you want the all off the massage showing befor you send your post, but otherwise is lloking good!